src/main/drivers/accgyro/accgyro.h

```diff
@@ -58,6 +58,7 @@ typedef enum {
     GYRO_ICM20689,
     GYRO_ICM42605,
     GYRO_ICM42688P,
+    GYRO_IIM42653,
```

Comment:
This will break all saved configs. Put it last in enum (it is also possible to increment PG version, but IMO appending is beter)